I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/minimart.rpy", line 38, in script
    v   "[name] ! Itu minimartnya ada di seberang jalan."
Exception: Sayer 'v' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/minimart.rpy", line 38, in script
    v   "[name] ! Itu minimartnya ada di seberang jalan."
  File "C:\Users\ASUS\Downloads\renpy-8.0.3-sdk.7z\renpy-8.0.3-sdk\renpy\ast.py", line 881, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\ASUS\Downloads\renpy-8.0.3-sdk.7z\renpy-8.0.3-sdk\renpy\ast.py", line 774, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'v' is not defined.

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.3.22090405
Simulasi Kehidupan Kuliah 1.0
Wed Dec 28 01:35:08 2022
